app/src/main/java/com/example/atheneum/models/Transaction.java

```diff
@@ -153,16 +171,11 @@ public boolean getOScan() {
     /**
      * @param oScan New value of oScan
      */
+    @Exclude

```

Comment:
Why the exclude?

Reply:
Without it I was getting errors that firebase found conflicting setters

Reply:
That's because the property name was wrong for this method it should be oScan instead of bScan

Reply:
Shoot my bad, fixed it now

Reply:
No worries, I think that was there from when the Transaction class was first written.